plugins/device/init.go

```diff
@@ -32,7 +32,17 @@ func (p *Plugin) RestRoutes() route.Routes {
 			Version:      1,
 			RequestType:  utils.GetTypeString((*deviceapi.AddDeviceReq)(nil)),
 			ResponseType: utils.GetTypeString((*deviceapi.AddDeviceResp)(nil)),
-			HandlerFunc:  deviceAddHandler},
+			HandlerFunc:  deviceAddHandler,
+		},
+		route.Route{
+			Name:         "PeerEditZone",
+			Method:       "POST",
+			Pattern:      "/peers/{peerid}/zone",
+			Version:      1,
+			RequestType:  utils.GetTypeString((*deviceapi.PeerEditZoneReq)(nil)),
+			ResponseType: utils.GetTypeString((*deviceapi.PeerEditZoneResp)(nil)),
+			HandlerFunc:  peerEditZoneHandler,
```

Comment:
This need not be separate API. It can be part of peer edit request, now that `Zone` is part of peer create request.